native-activity/app/build.gradle

```diff
@@ -0,0 +1,42 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion = 23
+    buildToolsVersion = '23.0.3'
+
+    defaultConfig {
+        applicationId = 'com.example.native_activity'
+        minSdkVersion   9
+        targetSdkVersion  23
+        cmake {
+            abiFilters 'x86', 'x86_64', 'armeabi', 'armeabi-v7a', 'arm64-v8a'
+        }
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+    externalNativeBuild {
+        cmake {
+            path 'src/main/cpp/CMakeLists.txt'
+        }
+    }
+    // Documentation purpose: app/src/main/jni is reserved for deprecated native build system;
+    // Steer away from this directory by DELETING it for ndk-build/cmake + android studio usage.
+    // Otherwise you need to
+    //    add android.useDeprecatedNdk=true to gradle.properties
```

Comment:
Please remove suggest to usDeprecatedNdk.  Even though that's what the error message may seem to suggest, you should never add android.useDeprecatedNdk=true to gradle.properties when using external build.  It will enable the old NDK integration.